roles/azimuth/defaults/main.yml

```diff
@@ -152,6 +152,13 @@ azimuth_authenticator_federated_identity_providers:
     provider: "{{ azimuth_authenticator_federated_provider }}"
     protocol: "{{ azimuth_authenticator_federated_protocol }}"
 #   List of authenticators in the order in which they should appear in the form
+
+# Additional password authenticator variables, set azimuth_authenticators_extra
+# in Azimuth variables config and set all the required variables for the authenticator
+# in a similar fashion to those below in azimuth_authenticators.
+azimuth_authenticators_extra: >-
+  []
+
```

Comment:
I think an example here is more clear than a comment, similar to the curated sizes variable earlier in this file.

```suggestion

# Additional authenticators to include
azimuth_authenticators_extra: []
# Example for authenticating with an additional Keystone domain:
#    - hidden: false
#       label: "What shows up on login dropdown menu"
#       name: a_unique_name
#       type: openstack-password
#       openstackPassword:
#         authUrl: https://keystone.openstack.example.com/v3
#         domain: my-domain
#         verifySsl: true
```